Sources/Corvus/Endpoints/Modifiers/AuthModifier.swift

```diff
@@ -8,7 +8,7 @@ public final class AuthModifier<
     Q: QueryEndpoint,
     E: EagerLoadable
 >: QueryEndpoint
-where E.EagerLoadValue: CorvusUser {
+where E: CorvusUser, Q.QuerySubject == E.From {
```

Comment:
When I test this with the [example project](https://github.com/Apodini/corvus-example-project) (branch support-vapor4-rc), I get a bunch of errors which I think have to do with this change. What was the reason for it?

I think `E.From` needs to be constrained to `CorvusUser` too, if it is done like this? See if you can reproduce with the example project.